examples/zero_token_operations/main.go

```diff
@@ -114,10 +123,11 @@ func main() {
 		panic(err)
 	}
 
-	_, err = helper.ExecuteSteps(0, 5, client)
-	if err != nil {
-		panic(fmt.Sprintf("%v : error in helper", err))
-	}
+	// TODO there is currently possible bug in services causing this operation to fail, should be investigated
```

Comment:
Maybe we need to add a test for this and keep it monitored.

Reply:
I will create issue about this once i've investigated it.